spec/controllers/announcements_controller_spec.rb

```diff
@@ -55,6 +56,31 @@
     end
   end
 
+  describe "DELETE /coronavirus/:coronavirus_page_slug/announcements/:id" do
+    before do
+      stub_user.permissions << "Unreleased feature"
+      setup_github_data
+      stub_coronavirus_publishing_api
+    end
+
+    let(:announcement_params) do
+      {
+        id: announcement,
+        coronavirus_page_slug: coronavirus_page.slug,
+      }
+    end
+
+    subject { delete :destroy, params: announcement_params }
+
+    it "redirects to the coronavirus page on success" do
```

Comment:
I tried this and it didn't work... for some reason the announcement seems to need to be created before the `subject { delete :destroy, params: announcement_params }` is called. I moved the `announcement` let within the `delete` test so that makes it slightly more readable, still not ideal!